sdk/python/lib/test/langhost/resource_op_bad_inputs/test_resource_op_bad_inputs.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2016-2018, Pulumi Corporation.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from os import path
+from ..util import LanghostTest
+
+
+class UnhandledExceptionTest(LanghostTest):
+    def test_unhandled_exception(self):
+        self.run_test(
+            program=path.join(self.base_path(), "resource_op_bad_inputs"),
+            expected_error="Program exited with non-zero exit code: 1")
```

Comment:
I don’t quite follow how this test tests the change here - didn’t we fail before as well (just with a bad error message)?

Reply:
We don't really have a great way to validate the precise error we raise in the integration test framework. Let me see if we have unit tests somewhere.

Reply:
Added a unit test.